go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	go.uber.org/atomic v1.3.1
 	go.uber.org/zap v1.7.1
-	golang.org/x/net v0.0.0-20200822124328-c89045814202
+	golang.org/x/net v0.3.0
```

Comment:
Changes to go.mod affect Yandex's internal automation. 
Could you undo the changes in these files for this PR